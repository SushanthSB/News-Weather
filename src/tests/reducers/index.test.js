import renderer from 'react-test-renderer';
import Reducers from '../../reducers/';

describe('Reducers()',() => {
	it('Should not be modified',() => {
		expect(Reducers).toMatchSnapshot();
	})
})