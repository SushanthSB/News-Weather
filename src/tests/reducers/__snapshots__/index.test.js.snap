// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reducers() Should not be modified 1`] = `[Function]`;
